.github/workflows/valgrind.yml

```diff
@@ -0,0 +1,51 @@
+name: valgrind
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "jq-*"
```

Comment:
We don't need to trigger on tags; all commits in master branch is already tested above.

Reply:
Running CI on tags is important. When we create a release tag, the CI workflow runs twice for the branch and tag of the same commit. But since `github.ref` is different (`refs/tags/jq-1.X` vs `refs/heads/master`), release (and docker) job(s) only run on the tag.

Reply:
Yeah, I just read `ci.yml` and noticed that it uses that to upload a release after a tag that starts with `jq-` is created; thank you